core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -30,18 +40,26 @@
 
     public static final String NAME = "missing";
 
-    private String name;
+    public static final boolean DEFAULT_NULL_VALUE = false;
 
-    private Boolean nullValue;
+    public static final boolean DEFAULT_EXISTENCE_VALUE = true;
 
-    private Boolean existence;
+    private String fieldPattern;
+
+    private Boolean nullValue = DEFAULT_NULL_VALUE;
+
+    private Boolean existence = DEFAULT_EXISTENCE_VALUE;
 
     static final MissingQueryBuilder PROTOTYPE = new MissingQueryBuilder(null);
 
-    public MissingQueryBuilder(String name) {
-        this.name = name;
+    public MissingQueryBuilder(String fieldPattern) {
+        this.fieldPattern = fieldPattern;
```

Comment:
i'm happy to add the check, but do we do it also for the value of TermQueryBuilder?